### PR TITLE
ENH: update CI configuration

### LINKED ISSUE #33: Simplify `wrapping/itkSplitComponentsImageFilter.wrap`
When a new ITK  build package is generated and contains the CMake variable `WRAP_ITK_SCALAR_IMAGE_PIXEL_TYPES`, update `wrapping/itkSplitComponentsImageFilter.wrap` to use this variable.

### LINKED ISSUE #41: Python Wrapping errors with ITK v5.1.1
We are unable to run Python Wrapping for `ITKSplitComponents` without wrapping errors. The terminal output is provided below when ran in the following conditions:

- Ubuntu 18.04.5
- ITK v5.1.1
- Python 3.5.9

Screen Shot 2020-08-26 at 9 56 54 AM

These errors are consistent amongst MacOS and Windows builds as well. I have provided a link to all 3 failing builds [here](https://github.com/mseng10/ITKSplitComponents/runs/1032001827?check_suite_focus=true). This issue will require the following changes:
- [ ] Fix Python Wrapping for Ubuntu, Windows and MacOS.
- [ ] Create PR containing wrapping changes and uncomment Python Package Builds in GitHub Actions CI with `TODO` removed. Verify all Python Packages build correctly.
- [ ]  Create an API token on PyPI for the package, and put in the repository Settings -> Secrets under the key pypi_password.
- [ ] Create new Git tag, either locally and pushing, or in the GitHub web Release page, this will trigger builds and push the packages to PyPI.

### PR DESCRIPTION
**Pull request by dzenanz**
Closes #33. Closes #41.

The error messages were:
```text
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 2 >, 2 >, itk::Image< double, 2 > >
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 3 >, 3 >, itk::Image< double, 3 > >
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 4 >, 4 >, itk::Image< double, 4 > >
```

### CODE REVIEW

**Comment by dzenanz** [comment]
Using WRAP_ITK_VECTOR_IMAGE_PIXEL_TYPES instead of WRAP_ITK_SCALAR_IMAGE_PIXEL_TYPES did not help. The error message is still the same:
```text
[10/16] Generating ../../../../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i, ../../../../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilterSwigInterface.h, ../../../../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.idx, ../../../../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/Python/itk/Configuration/SplitComponents_snake_case.py
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 2 >, 2 >, itk::Image< double, 2 > >
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 3 >, 3 >, itk::Image< double, 3 > >
itkSplitComponentsImageFilter: warning(4): ITK type not wrapped, or currently not known: itk::ImageToImageFilter< itk::Image< itk::Vector< double, 4 >, 4 >, itk::Image< double, 4 > >
[11/16] Generating itkSplitComponentsImageFilterPython.cpp, ../../../../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/Python/itk/itkSplitComponentsImageFilterPython.py
FAILED: Wrapping/Modules/SplitComponents/itkSplitComponentsImageFilterPython.cpp ../../../ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/Python/itk/itkSplitComponentsImageFilterPython.py 
cd /work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/python && /work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/SwigInterface/swig/bin/swig -c++ -python -O -features autodoc=2 -py3 -doxygen -Werror -w302 -w303 -w312 -w314 -w361 -w362 -w350 -w383 -w384 -w389 -w394 -w395 -w467 -w508 -w509 -o /work/_skbuild/linux-x86_64-3.7/cmake-build/Wrapping/Modules/SplitComponents/itkSplitComponentsImageFilterPython.cpp -I/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/SwigInterface/swig/share/swig/4.0.2/python -I/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/SwigInterface/swig/share/swig/4.0.2 -I/work/ITK-source/ITK/Wrapping/Generators -I/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/python -I/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs -outdir /work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Generators/Python/itk /work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:84: Warning 401: Nothing known about base class 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,2 >,2 >,itk::Image< double,2 > >'. Ignored.
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:84: Warning 401: Maybe you forgot to instantiate 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,2 >,2 >,itk::Image< double,2 > >' using %template.
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:163: Warning 401: Nothing known about base class 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,3 >,3 >,itk::Image< double,3 > >'. Ignored.
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:163: Warning 401: Maybe you forgot to instantiate 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,3 >,3 >,itk::Image< double,3 > >' using %template.
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:242: Warning 401: Nothing known about base class 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,4 >,4 >,itk::Image< double,4 > >'. Ignored.
/work/ITK-cp37-cp37m-manylinux2014_x64/Wrapping/Typedefs/itkSplitComponentsImageFilter.i:242: Warning 401: Maybe you forgot to instantiate 'itk::ImageToImageFilter< itk::Image< itk::Vector< double,4 >,4 >,itk::Image< double,4 > >' using %template.
ninja: build stopped: subcommand failed.
  File "/opt/python/cp37-cp37m/lib/python3.7/site-packages/skbuild/setuptools_wrap.py", line 551, in setup
    cmkr.make(make_args, env=env)

  File "/opt/python/cp37-cp37m/lib/python3.7/site-packages/skbuild/cmaker.py", line 482, in make
    os.path.abspath(CMAKE_BUILD_DIR)))
```

**Comment by dzenanz** [comment]
Python packages deviate from general ITK wrapping defaults. Packages have these scalars: `SS;UC;US;F;D`. `US` and `D` are additions to usual defaults. I am now able to reproduce the error locally.

File: wrapping/itkSplitComponentsImageFilter.wrap [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
+    list(APPEND types "D")
+  endif()
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     foreach(t ${types})
       if(DEFINED ITKT_IV${t}${d}${d})
-      itk_wrap_template(
-        "${ITKM_IV${t}${d}${d}}${ITKM_I${t}${d}}"
-        "${ITKT_IV${t}${d}${d}}, ${ITKT_I${t}${d}}")
+        itk_wrap_template(
+          "${ITKM_IV${t}${d}${d}}${ITKM_I${t}${d}}"
```
Comment (thewtex) [inline wrapping/itkSplitComponentsImageFilter.wrap:15]: `CV` for CovariantVector, I believe.

We also want to wrap Vector.

**Comment by dzenanz** [comment]
It looks like [files.pythonhosted.org](files.pythonhosted.org) is experiencing issues right now.

**Pull request merged**